doc/ko/시작하기.txt

```diff
@@ -0,0 +1,139 @@
+이 문서는 영어 README 파일의 일부 번역본입니다. 
```

Comment:
This filename is not portable, and I am afraid, it could lead to some problems. Could you please rename it to ASCII-only name?